### ISSUE TITLE
Terminal & server integration

### ISSUE THREAD

**Issue by TobiaszCudnik** [spiffcode/ghedit#16]
It would be great to be able to test your code straight from the browser. SSH client inside of the integrated  @#terminal should be sufficient as a start, but avoiding commit-to-test is also highly desirable.

Anyway great stuff you have here, made my day!

**Comment by stubailo** [comment]
Yeah I am currently trying to find a way to code on an iPad and this seems like the best project to start with! Going to see if I can extend it with some kind of local file access on the remote machine.
